android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt

```diff
@@ -12,12 +12,14 @@ import io.mosip.tuvali.wallet.transfer.message.*
 import java.util.*
 import io.mosip.tuvali.transfer.Util.Companion.getLogTag
 
+const val MAXIMUM_RETRY_FRAME_LIMIT = 5;
```

Comment:
can change variable name to `MAX_FAILURE_FRAME_RETRY_LIMIT`

Reply:
Oh, this change has already been ACK-ed by you it seems